production-dependency-templates/exocom/dependency.tf

```diff
@@ -0,0 +1,39 @@
+module "exocom_cluster" {
+  source = "github.com/Originate/exosphere.git//terraform//aws//dependencies//exocom//exocom-cluster?ref={{terraformCommitHash}}"
+
+  availability_zones      = "${module.aws.availability_zones}"
```

Comment:
We should add comments to the top level readme about what vars are passed into the terraform files. This can done later but lets add an empty section with a TODO tag in it.

Reply:
Examples of non-generic keys that come in here are 
* `var.exocom_docker_image` - available because `docker-image` is specified in the `config.yml` 
* `var.exocom_env_vars` - which we could say are available because docker-images

In the docs: exocom would be replaced with <id> (the key in the dependencies block)